### DISCUSSION TITLE
Issues pushing work to git repo

### DISCUSSION THREAD

**Discussion by benedwards195** [community/community#76879] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am trying to push a uni project I worked on into one of my git repositories. When trying to push, the terminal server just stops (please see attached screenshot). What should I do about this?
<img width="494" alt="Screenshot 2023-11-24 at 15 07 47" src="https://github.com/community/community/assets/9057647/1d7bf765-cc52-41dc-b572-3224859c1f85">

**Comment by ibtissammassa** [comment]
Hey there! From the provided information, it seems like there might be a hiccup with the push process.

To address this, first, let's ensure your Git configurations are correct, especially the remote origin URL. You can verify this by running `git remote -v` to confirm that the remote URLs are accurate.

If it is, then if your repository contains a large number of files or sizable files, it might be causing the push to stall. Consider pushing smaller chunks or breaking down commits to make the process smoother.

Give these steps a try, and if you're still encountering issues, please update me with more details. I'm here to assist and troubleshoot further

**Reply by benedwards195** [reply]
Hey,

It turns out that I needed to reconfigure it. Having generated a new PAT for it as well, I now receive this problem. Would this suggest that my file is too big? As to be fair, I am trying to push an entire project into one commit, which git maybe doesn't have the capacity for
<img width="572" alt="Screenshot 2023-11-24 at 20 58 05" src="https://github.com/community/community/assets/9057647/00c4aaeb-df0d-4e6e-9f9d-a518a7090808">
